plugins/postcss-nesting/CHANGELOG.md

```diff
@@ -1,11 +1,10 @@
 # Changes to PostCSS Nesting
 
-⚠️ We will most likely not finish this release and switch over to the plugin being worked on in the experimental track.
```

Comment:
Because the specification will not be stable in time we will continue using this plugin for a bit longer.